maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -206,6 +206,20 @@ data class TapOnPointCommand(
     }
 }
 
+data class TapOnPercentCommand(
+    val percentX: Int,
```

Comment:
I will make some tweaks here myself - those need to be `String` values to allow users to inject Javascript in there (in `evaluateScripts`)